### PR TITLE
fix(ovm): Fix implementation of IsContainedDecider

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

**Comment by tkmct** [comment]
could you change `.raw` to `.data`?

**Comment by syuhei176** [comment]
> could you change `.raw` to `.data`?

sure. modified.

File: src/ovm/deciders/atomic/IsContainedDecider.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ export class IsContainedDecider implements Decider {
 
     return {
       outcome:
-        first.start.raw >= second.start.raw &&
-        first.end.raw <= second.start.raw,
```
Comment (tkmct) [inline src/ovm/deciders/atomic/IsContainedDecider.ts]: is this right?
I thought this decider means first is contained by second.
so, `first.start >= second.start` and `first.end <= second.end`

Comment (syuhei176) [reply src/ovm/deciders/atomic/IsContainedDecider.ts]: Yes, So I changed `second.start` to `second.end` following your comment.

**Pull request merged**